### PR TITLE
feat: deployment-target view

### PR DESCRIPTION
**Pull request by benPearce1**
[sc-6874]

depends on https://github.com/OctopusDeploy/go-octopusdeploy/pull/161

feat: deployment-target azure-web-app view
feat: deployment-target cloud-region view
feat: deployment-target listening-tentacle view
feat: deployment-target ssh view
feat: deployment-target polling-tentacle view

azure web app:
image

offline drop:
image

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,6 +2,10 @@ module github.com/OctopusDeploy/cli
 
 go 1.19
 
+replace (
+	github.com/OctopusDeploy/go-octopusdeploy/v2 => ../go-octopusdeploy
+)
+
```
Comment (jbristowe) [inline go.mod:8]: We should remove this

Comment (benPearce1) [reply go.mod:8]: this is waiting on https://github.com/OctopusDeploy/go-octopusdeploy/pull/161 to be merged

Comment (benPearce1) [reply go.mod:8]: and then it will be removed

**Comment by benPearce1** [comment]
This has been updated with a table output for all `deployment-target view` commands

**Review by jbristowe — APPROVED** [review]

**Pull request merged**